src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -82,10 +82,24 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
         val result = driver.executeScript("""
             try {
                 window._jibriParticipants = [];
+                const existingMembers = APP.conference._room.room.members || {};
+                console.log("There were " + Object.keys(existingMembers).length + " existing members");
+                for (let key in existingMembers) {
```

Comment:
To be cute...you already did Object.keys so you might as well store it and do a forEach/reduce/filter/map

Reply:
though as much as i'd like to do the map/filter/etc i want the debug printing in the 'else' cases to know what we're filtering out so will just do a loop.